src/codegen.rs

```diff
@@ -142,6 +142,17 @@ macro_rules! app {
             [ $( $pattern => $queue ),* ],
         );
     };
+    (
+        $(broker =)? REDIS { $broker_url:expr },
+        $(tasks =)? [ $( $t:ty ),* $(,)? ],
+        $(task_routes =)? [ $( $pattern:expr => $queue:expr ),* $(,)? ]
+    ) => {
+        $crate::__app_internal!(
+            $crate::broker::RedisBroker { $broker_url },
+            [ $( $t ),* ],
+            [ $( $pattern => $queue ),* ],
+        );
+    };
```

Comment:
`$(broker =)? $broker:ty  { $broker_url:expr }` creates the error: `local ambiguity: multiple parsing options: built-in NTs ty ('broker') or 1 other option.` 

However following works: `broker = $broker:ty  { $broker_url:expr }`.

Reply:
Hmm darn. Ok, well I realized we could express that macro a single way that deals with both cases for the trailing comma: https://github.com/rusty-celery/rusty-celery/pull/185.

With that, I'm fine with just repeating the single pattern like you have it for each new broker. But I'd prefer to use `Redis` instead of `REDIS`.